src/open_inwoner/openzaak/models.py

```diff
@@ -448,6 +448,15 @@ def form_service(self, service):
         ),
         default=False,
     )
+    relevant_role = models.CharField(
```

Comment:
The naming feels a bit opaque, especially when you're out of context (relevant for what?). What about:

```suggestion
    limit_user_visible_cases_to_role = models.CharField(
```
Bit more verbose, but perhaps more readable and explicit in the long run.